src/stores/cluster/index.js

```diff
@@ -33,77 +30,57 @@ export default class ClusterStore extends Store {
   @observable selectedRowKeys = [];
   @observable clusterIds = [];
 
-  @observable currentNodePage = 1;
-  @observable searchNode = '';
-  @observable selectNodeStatus = '';
-
-  @observable keyPairs = [];
-  @observable pairId = '';
-  @observable currentPairId = '';
-  @observable name = '';
-  @observable pub_key = '';
-  @observable description = '';
   @observable env = '';
   @observable versionId = '';
 
-  page = 'index';
-
   // cluster job queue
   @observable
   jobs = {
     // job_id=> cluster_id
   };
+
   store = {};
 
-  @action.bound
+  @action
   showModal = type => {
     this.modalType = type;
     this.isModalOpen = true;
   };
 
-  @action.bound
+  @action
   hideModal = () => {
+    this.modalType = '';
     this.isModalOpen = false;
   };
 
   @action
-  fetchAll = async (params = {}) => {
-    let defaultParams = {
-      sort_key: 'create_time',
-      limit: this.pageSize,
-      offset: (this.currentPage - 1) * this.pageSize,
-      status: this.selectStatus ? this.selectStatus : this.defaultStatus
-    };
-
-    if (params.noLimit) {
-      defaultParams.limit = this.maxLimit;
-      defaultParams.offset = 0;
-      delete params.noLimit;
-    }
-
+  fetchAll = async params => {
+    params = this.normalizeParams(params);
     if (this.searchWord) {
-      defaultParams.search_word = this.searchWord;
+      params.search_word = this.searchWord;
     }
     if (this.appId) {
-      defaultParams.app_id = this.appId;
+      params.app_id = this.appId;
     }
     if (this.runtimeId) {
-      defaultParams.runtime_id = this.runtimeId;
+      params.runtime_id = this.runtimeId;
     }
     if (this.userId) {
-      defaultParams.owner = this.userId;
+      params.owner = this.userId;
     }
 
     this.isLoading = true;
-    const result = await this.request.get('clusters', assign(defaultParams, params));
+    const result = await this.request.get('clusters', params);
```

Comment:
If the parameters has app_id, runtime_id, owner, the store appId, runtimeId, userId are not null, these will cover the passed parameters.But now there should be no such code.

Reply:
you can check `stores/Store.js` which will filter `null` and `undefined` param